easybuild/easyconfigs/i/IMB/IMB-4.1-intel-2017.02.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'MakeCp'
+
+name = 'IMB'
+version = '4.1'
+
+homepage = 'https://software.intel.com/en-us/articles/intel-mpi-benchmarks'
+description = """The Intel MPI Benchmarks perform a set of MPI performance measurements for point-to-point and
+ global communication operations for a range of message sizes"""
+
+toolchain = {'name': 'intel', 'version': '2017.02'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://software.intel.com/sites/default/files/managed/34/aa/']
+sources = ['%(name)s_%(version)s.tgz']
+
+prebuildopts = "cd src && "
+buildopts = "all"
```

Comment:
using buildopts = 'all' in foss doesn't seem work because IMB expects mpiicc by default. 
`FAILED: Installation ended unsuccessfully (build directory: /dev/shm/IMB/4.1/foss-2017a): build failed (first 300 chars): cmd "export MPI_HOME=$EBROOTOPENMPI &&  make -j 1 all" exited with exitcode 2 and output:
gmake -f make_ict IMB-EXT`